app/utils/callback/lava.py

```diff
@@ -438,6 +440,11 @@ def _add_test_results(group, suite_results, suite_name):
             models.TIME_KEY: "0.0",
         }
         test_case.update({k: test[v] for k, v in TEST_CASE_MAP.iteritems()})
+        try:
+            test_case[models.INDEX_KEY] = dateutil.parser.parse(test_case[models.INDEX_KEY])
```

Comment:
I think it would be a bit better to have the index follow the test case number i.e. 1, 2, 3... rather than a random integer derived from the timestamp.

Reply:
Yes, I have the fix done already. Testing it on a local env.